.github/workflows/hydrate-universe.yml

```
@@ -0,0 +1,182 @@
+  schedule:
+    # 21:30 UTC = 17:30 ET, ~75 min after the 16:00 ET close. Weekdays only:
+    # a weekend run would re-card Friday's bars and burn API calls to write
+    # rows `shouldReplaceCard` correctly refuses to replace.
+    - cron: '30 21 * * 1-5'
+  workflow_dispatch:
+    inputs:
+      universe:
+        description: 'Which lane to hydrate (blank = stock then etf)'
+        required: false
+        type: choice
+        options: ['', 'stock', 'etf']
```

actionlint (1.7.12)

[error] string should not be empty

(syntax-check)